Unity-Technologies/ScriptableRenderPipeline
# Fix Case 1131646: IsFrontFace Breaks Subgraphs

### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?

---
Fixing an error that caused `IsFrontFace` nodes not to compile from within Sub Graph nodes.
### Release Notes
Please add any useful notes about the feature/fix that might be helpful for others.

---
Added `IMayRequireFaceSign` to Subgraph node.
### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?
Pass

**Manual Tests**: What did you do?
- Other: 
Created multiple sub graph assets referencing `IsFaceSign` to confirm the errors are gone. 
**Automated Tests**: What did you setup?

Any test projects to go with this to help reviewers?

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High?
Low
**Halo Effect**: None, Low, Medium, High?
Low
---
### Comments to reviewers
Notes for the reviewers you have assigned.
